query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/fields/config/LdapDirectorySettingsField.java

```diff
@@ -150,83 +135,66 @@ public String memberAttributeReferencedInGroup() {
         return memberAttributeReferencedInGroup.getValue();
     }
 
-    public LdapSettingsField useDefaultAuthentication() {
-        useCase.isRequired(true);
+    public LdapDirectorySettingsField useDefaultUser() {
```

Comment:
This method should go away if we are only passing what is required for testAttributeMapping